controllers/goharbor/harbor/chartmuseum.go

```diff
@@ -87,10 +87,14 @@ func (r *Reconciler) GetChartMuseum(ctx context.Context, harbor *goharborv1alpha
 
 	tls := harbor.Spec.InternalTLS.GetComponentTLSSpec(r.GetInternalTLSCertificateSecretName(ctx, harbor, harbormetav1.ChartMuseumTLS))
 
+	annotations := make(map[string]string)
+	annotations[harbormetav1.NetworkPoliciesAnnotationName] = harbormetav1.NetworkPoliciesAnnotationDisabled
+
 	return &goharborv1alpha2.ChartMuseum{
 		ObjectMeta: metav1.ObjectMeta{
-			Name:      name,
-			Namespace: namespace,
+			Name:        name,
+			Namespace:   namespace,
+			Annotations: annotations,
```

Comment:
This map is static (no elements is added in `if` statements), so please initialize it directly in the structure.

Same number of line (talking about the `// nolint:funlen` that is okey for me).
IMO it is easier to understand since it is easy to find all annotations.

```suggestion
			Annotations: map[string]string{
				harbormetav1.NetworkPoliciesAnnotationName: harbormetav1.NetworkPoliciesAnnotationDisabled,
			},
```